tensorflow_cloud/deploy.py

```diff
@@ -167,8 +167,16 @@ def _stream_logs(job_id):
     """
     try:
         print('Streaming job logs: ')
-        subprocess.run(
-            ["gcloud", "ai-platform", "jobs", "stream-logs", job_id])
+        process = subprocess.Popen(
+            ['gcloud', 'ai-platform', 'jobs', 'stream-logs', job_id],
+            stdout=subprocess.PIPE)
+        while True:
+            output = process.stdout.readline()
+            # Break out of the loop when poll returns an exit code.
+            if process.poll() is not None:
+                break
+            if output:
+                print(output.decode().replace('\x08', ''))
```

Comment:
I wonder if we could reproduce the same behavior as what happens in a terminal or a notebook output cell (with the last progbar line replacing the previous line):

See in CS: tensorflow/python/keras/utils/generic_utils.py;l=566-567

This would involve replacing `\x08` with `\b`, I presume.

Reply:
Good point. Discussed offline, i will experiment with this and send a separate PR.